client/src/entrypoints/admin/bulk-actions.js

```diff
@@ -1,11 +1,12 @@
 /* global wagtailConfig */
 
-const BULK_ACTION_PAGE_CHECKBOX_INPUT = 'bulk-action-checkbox';
-const BULK_ACTION_SELECT_ALL_CHECKBOX_TH = 'bulk-actions-filter-checkbox';
-const BULK_ACTION_FILTERS_CLASS = `${BULK_ACTION_SELECT_ALL_CHECKBOX_TH} .c-dropdown__item a`;
-const BULK_ACTION_CHOICES_DIV = 'bulk-actions-choices';
-const BULK_ACTION_NUM_OBJECTS_SPAN = 'num-objects';
-const BULK_ACTION_NUM_OBJECTS_IN_LISTING = 'num-objects-in-listing';
+const BULK_ACTION_PAGE_CHECKBOX_INPUT = '[data-bulk-action-checkbox]';
+const BULK_ACTION_SELECT_ALL_CHECKBOX = '[data-bulk-action-select-all-checkbox]';
+const BULK_ACTIONS_CHECKBOX_PARENT = '[data-parent-id]';
```

Comment:
suggestion: namespace them this and the vars below with `data-bulk-actions-` to ensure there is no clash with other elements (e.g. from packages, or possibly future core ones)

Reply:
Should this be addressed in this pr, or a future pr?